layouts/shortcodes/friday_hack_header.html

```diff
@@ -1,7 +1,9 @@
 <blockquote>
     <strong>Date/Time</strong>: Friday, {{ .Get "date" }}{{ if not (in (.Get "date") " at ") }} at 7:00pm{{ end }}<br />
     <strong>Venue</strong>: {{ .Get "venue" | safeHTML }} <br />
-    <strong>Free {{ .Get "food" | default "pizza" }} is served before the talks at 6:30pm!</strong> <br />
+    {{ with .Get "food" }}
+      <strong>Free {{ . }} is served before the talks at 6:30pm!</strong> <br />
+    {{ end }}
```

Comment:
Wild idea, i wonder how much more we can automate the youtube link and remove the food info after the event is over.

Reply:
The food info is fine but youtube link meaning it needs to scrape or have youtube API, which um slightly mafan